Common/Core/TrackSelectorPID.h

```diff
@@ -464,23 +471,29 @@ class TrackSelectorPidBase
   template <typename T>
   bool isElectronAndNotPion(const T& track, bool useTof = true, bool useRich = true)
   {
+    static constexpr float PidNsigmaInvalid = -1000.f;
+    static constexpr float PidTofRichTransitionPMin = 0.4f;
+    static constexpr float PidTofRichTransitionPMax = 0.6f;
+    static constexpr float PidRichPionBandPMin = 1.0f;
+    static constexpr float PidRichPionBandPMax = 2.0f;
```

Comment:
```suggestion
    static constexpr float NSigmaInvalid{-1000.f};
    static constexpr float PTofRichTransitionMin{0.4f};
    static constexpr float PTofRichTransitionMax{0.6f};
    static constexpr float PRichPionBandMin{1.0f};
    static constexpr float PRichPionBandMax{2.0f};
```
Please
- always start the name with the quantity,
- don't use `pid`; as it's redundant, since the whole class is for PID and it's clear from the name too.